DSharpPlus/Net/Models/GuildEditModel.cs

```diff
@@ -36,6 +37,7 @@ public class GuildEditModel : BaseEditModel
         /// <summary>
         /// The new guild voice region.
         /// </summary>
+        [ObsoleteAttribute("Channels use their own voice regions. Guild region is deprecated. Please use the newer one on the voice channels.", false)]
```

Comment:
Write [Obsolete] instead of [ObsoleteAttribute]

Reply:
Done.